### PR TITLE
Material theme

### PR DESCRIPTION
**Pull request by tulioag**
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-login/61)

### CODE REVIEW

File: demo/i18n-demos.html [hunk 1]
```diff
@@ -11,15 +11,15 @@ <h3>Internationalization</h3>
     <p>Set the <code>i18n</code> object to localize the component</p>
     <vaadin-demo-snippet id="login-demo-internationalisation">
       <template preserve-content>
-        <style include="lumo-typography lumo-color">
+        <style>
```
Comment (web-padawan) [inline demo/i18n-demos.html]: this empty style block and closing `<custom-style>` tag below should be removed.

File: src/vaadin-login-overlay-wrapper.html [hunk 2]
```diff
@@ -15,28 +15,6 @@
         outline: none;
       }
 
-      [part="card"] {
```
Comment (web-padawan) [inline src/vaadin-login-overlay-wrapper.html]: is this change necessary after all? consider reverting it, so that PR does not change neither `src` not Lumo

File: theme/material/vaadin-login-overlay-styles.html [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+      [part="brand"] h1 {
+        color: inherit;
+        margin: 0;
+      }
+
+      [part="backdrop"] {
+        background: var(--material-background-color);
+      }
+
+      vaadin-login {
```
Comment (web-padawan) [inline theme/material/vaadin-login-overlay-styles.html]: this selector is not targeting anything (as the login is slotted), and should be removed.
the ` [part="form"] ::slotted(*) {` selector below is a proper one.

File: theme/material/vaadin-login-overlay-styles.html [hunk 4]
```diff
@@ -0,0 +1,356 @@
...
+        background-color: #f2f4f6;
+      }
+
+      [part="form"] ::slotted(*) {
+        position: relative;
+        bottom: 85px;
+        margin: auto;
+      }
+
+      [part="form"] h2 {
```
Comment (web-padawan) [inline theme/material/vaadin-login-overlay-styles.html]: this selector is not targeting anything, and should be removed. 
the proper one is already defined in the `material-login-overlay`:
```
:host([theme~="with-overlay"]) [part="form"] h2
```

File: theme/material/vaadin-login-overlay-styles.html [hunk 5]
```diff
@@ -0,0 +1,356 @@
...
+        text-align: center;
+        font-weight: 500;
+      }
+
+      [part="form"] vaadin-button {
```
Comment (web-padawan) [inline theme/material/vaadin-login-overlay-styles.html]: this selector is not targeting anything, and should be removed. 
the proper one is already defined in the `vaadin-login-forgot-password-button` style module.

File: theme/material/vaadin-login-overlay-styles.html [hunk 6]
```diff
@@ -0,0 +1,356 @@
...
+
+        [part="card"],
+        [part="overlay"] {
+          width: 100%;
+          border-radius: 0;
+          box-shadow: none;
+          margin: 0;
+        }
+
+        [part="form"] h2 {
```
Comment (web-padawan) [inline theme/material/vaadin-login-overlay-styles.html]: same as above, selector can be removed as it is not correct.

File: demo/i18n-demos.html (hunk 1 above)
Comment (tulioag) [reply demo/i18n-demos.html]: Done
